delta/config/config.py

```diff
@@ -181,6 +181,7 @@ def load(self, yaml_file: str = None, yaml_str: str = None):
         Loads a config file, then updates the default configuration
         with the loaded values.
         """
+        print("Loading config file: " + yaml_file)
```

Comment:
Remove

Reply:
I found these were useful to have and honestly given the large amount of printing normally in TF I don't think they gum things up too much.  If there is some way to suppress all the TF messages I could see it making a difference.

Reply:
export TF_MIN_CPP_LOG_LEVEL=0 (or something like that) will suppress most of the TF messages.

Reply:
I ran into problems trying to add a verbose flag so I just commented the lines and will uncomment them on my work machines.